java/runtime/src/main/java/io/ray/runtime/config/RayConfig.java

```diff
@@ -121,7 +122,8 @@ public RayConfig(Config config) {
     }
 
     // Namespace of this job.
-    namespace = config.getString("ray.job.namespace");
+    String localNamespace = config.getString("ray.job.namespace");
+    namespace = StringUtils.isEmpty(localNamespace) ? UUID.randomUUID().toString() : localNamespace;
```

Comment:
Does the pattern of generated namespace string match Python?

Reply:
One example in java: efd2c1ea-f5b5-46b8-b1c3-3020fe2bfd70

Reply:
It's  UUID4 as well.